app.py

```diff
@@ -0,0 +1,34 @@
+from flask import Flask
+from flask_restx import Api
+from .config import config
+
+import os
+
+api = Api(
+    title="BlueSaver - MC426",
+    version="1.0",
+    doc="/",
+    description="Projeto com várias funcionalidades sobre a aplicação/uso da água na nossa sociedade",
+)
+
+def create_app(config_name: str = None):
+    app = Flask(__name__)
+    if config_name is None:
+        config_name = os.environ.get("FLASK_CONFIG", "production")
+
+    # Rotas das Features
+    # Graficos
+    from .features.graficos.views import graficos_ns

```

Comment:
Não seriam boas práticas fazer todas as importações necessárias no topo do arquivo?

Reply:
Não, de acordo com a documentação do "factory patterns" dos desenvolvedores do Flask, é boa prática utilizar a importação da rota dentro da função `create_app()`. Referências:
https://flask.palletsprojects.com/en/2.2.x/tutorial/views/
https://stackoverflow.com/questions/60142047/in-flask-is-it-possible-to-import-views-using-the-create-app-pattern-without-u
https://stackoverflow.com/questions/34823316/importing-views